release-notes/5.0/api-diff/netcoreapp3.1/.NET/standalone/5.0-standalone_System.Windows.Extensions.md

```diff
@@ -0,0 +1,122 @@
+# System.Windows.Extensions
+
+``` diff
+ {
+-    namespace System.Drawing {
```

Comment:
Not sure if we want a note, but these were all pushed down into System.Drawing.Conmmon.

Reply:
i added a comment for the entire assemblies to be moved down, i will add for these types as well